### DISCUSSION TITLE
Selected text in Drafts - highlight is barely visible

### DISCUSSION THREAD

**Discussion by dandusty** [dracula/dracula-theme#764] [Q&A] [answered]
Hi, I'm really enjoying the Dracula theme in Drafts, but I was wondering how I can request that the highlight color for selected text be changed to something more visible. On Mac and iPhone, whenever I select text I can barely see what I've selected.

**Comment by luxonauta** [comment, marked as answer]
Glad to hear that, @dandusty! 🥰

The ideal would be to create an Issue in [this repository](https://github.com/dracula/drafts), so the theme maintainer (@danielbayley) for this app might consider making this change.
